src/components/SharedComponents/Button/button.styles.ts

```diff
@@ -268,14 +275,15 @@ const buttonStyles = css<ButtonStylesProps>`
     props.variant === "buttonStyledAsLink" &&
     props.size === "medium" &&
     css`
-      & svg[name="underline-1"] {
-        top: 27px;
-        width: 96%;
+      & ${ButtonStyledAsLinkFocusUnderline} {
+        display: block;
+        bottom: -1px;
+        left: ${props.$iconPosition === "leading" ? "4px" : ""};
+        width: ${props.icon ? "calc(100% - 4px)" : "100%"};
       }
 
-      & div {
+      & ${Flex} {
         margin-left: 0;
-        font-weight: 600;
```

Comment:
I think this isn't required

Reply:
The designs have the font-weight as 600 in Figma, however I'm happy to leave this as 700 as it looks fine and would prevent us from adding further CSS for this exception

Reply:
I think if it can match the font weight in Figma (600), then that would be ideal as this will match the styles used elsewhere. I'm unsure of what the built-in styles of that specific button component are & how much work that is though

Reply:
Yup my mistake @assadk88 is correct

Reply:
@orangemug I have a question. 

In [`src/styles/utils/typography.ts`](https://github.com/oaknational/Oak-Web-Application/blob/main/src/styles/utils/typography.ts), the configurations `font-size: 16, font-weight: 600` (needed for the desktop version of the button as per Figma) and `font-size: 16, font-weight: 600` (needed for the mobile version of the button as per Figma) do not exist.

If we want to remain in alignment with the type scale already defined in the app we would have to choose from either:

```
  "body-2": [16, 24, 300, "-0.005em"],
  "body-3": [14, 20, 300, "-0.005em"],
  "body-2-bold": [16, 24, 700, "-0.005em"],
  "body-3-bold": [14, 20, 700, "-0.005em"],
```

but in all of these cases their font-weights are either too high or too low.

This is a long-winded way of saying, should I create a new configuration at font-weight 600 to match up with the Figma designs, or default to a pre-existing one? 

Adding custom CSS to `<ButtonAsLink/>` for font-weight would prevent the component from being implementation agnostic.

Reply:
Probably one for @HelenHarlow. What do you think is font weight `700` ok to keep inline with the other uses of this component?

Reply:
I've had a quick sweep through of the bold text on the website and it all appears to be 600. Basically, I'm happy to align to whatever bold text we're using in the app, but I'm not sure what the correct value is?

Reply:
@HelenHarlow So, further from what we discussed earlier:

* **Font size and weight**: Remove responsiveness from the font-size so it stays a consistent 16px (1rem) for mobile and desktop – hence this will allow us to retain the 600 font-weight
* **Stacking behaviour:** Have the banner stack at screen widths where it can no longer reasonably fit on one line

Having discussed the above with @orangemug , in order to function within the constraints of the DS/component library the stacking behaviour required a bit of custom CSS which would require adding another breakpoint to get the behaviour that you wanted, so we decided against it.

Instead we've decided to to have desktop display on one line, and have it stack as it hits tablet widths and below. Essentially we decided to stack much earlier. I've attached a video to show you what I mean.
I hope that's fine with you?

https://github.com/user-attachments/assets/485ddff3-6de3-4b04-a33d-aed06d1c7a1d